### ISSUE TITLE
Add support for text inputs in panels

### ISSUE THREAD

**Issue by dominikmayer** [jgthms/bulma#162]
I'd like to use a panel component for a checkbox list.

So far I couldn't find a way to properly style the text input. I would like it to fill the whole width and have the same style as inputs in content areas. (Wrapping it in a content block didn't help.)

Here's my code:

```html
  <section class="section">
    <div class="container">
      <div class="columns">
        <div class="column is-4">
          <nav class="panel">
            <p class="panel-heading">
              Checklist
            </p>
            <div class="panel-block">
              <form>
                <input type="text" />
              </form>
            </div>
            <label class="panel-checkbox">
              <input type="checkbox"> Remember me Remember me Remember me Remember me Remember me Remember me
            </label>
            <div class="panel-block">
              <button class="button is-primary is-outlined is-fullwidth">
                Finish all, close list
              </button>
            </div>
          </nav>
        </div>
      </div>
    </div>
  </section>
```

And the result:

![bildschirmfoto 2016-04-05 um 00 19 09](https://cloud.githubusercontent.com/assets/159476/14265831/23990858-fac4-11e5-8129-44da3d97044c.png)

**Comment by dominikmayer** [comment]
Strange: It seems to work in some cases. So after I constructed the example, I put it into a react app and got the following perfect output:

![bildschirmfoto 2016-04-05 um 00 33 20](https://cloud.githubusercontent.com/assets/159476/14266092/169f642e-fac6-11e5-8e06-8a5336e751d8.png)

This was with a content block. I then tried the same thing outside of react and it looked as before. Am I missing something here?
